src/requests/init.jl

```diff
@@ -108,9 +108,7 @@ function process(r::JSONRPC.Request{Val{Symbol("initialize")},InitializeParams},
         end
     end
 
-    response = JSONRPC.Response(r.id, InitializeResult(serverCapabilities))
-    send(response, server)
-    LanguageServer.send(LanguageServer.JSONRPC.Notification{Val{Symbol("julia/setStatus")},String}("server setting up"), server)
+    return InitializeResult(serverCapabilities)
```

Comment:
I'm removing the custom status for now, still hope to get the progress reporting up and running. If we can't figure that out, I'll put this back in.

Reply:
Shall we leave it in until it's replaced?

Reply:
I think it would have to go somewhere else, though, because it should really only be sent after the init sequence is over? Not sure. I'll put it back in tonight if I don't manage to get the progress reporting working this afternoon.